shlr/meson.build

```diff
@@ -2,7 +2,11 @@ res = run_command(py3_exe, '-c', '__import__("sys").exit(__import__("os").path.e
 if res.returncode() == 0
   message('Cloning capstone')
   git_cmd = 'clone -b next --depth 10 https://github.com/aquynh/capstone.git capstone'
-  run_command(git_exe, git_cmd.split())
+  if git_exe.found()
+    run_command(git_exe, git_cmd.split())
+  else
+    error('git is required to download shlr/capstone')
```

Comment:
And what about tarball?

Reply:
In the tarball `shlr/capstone` exists, so it should not reach this branch right?

Reply:
Who distributes this tarball?

Reply:
You brought up the tarball. So I assumed there is a source tarball bundling capstone.